packages/serialization/json/src/jsonSerializationWriter.ts

```diff
@@ -214,7 +254,7 @@ export class JsonSerializationWriter implements SerializationWriter {
 		}
 		this.writer.push(JSON.stringify(value), JsonSerializationWriter.propertySeparator);
 	};
-	private readonly writeAnyValue = (key?: string | undefined, value?: unknown | undefined): void => {
+	private readonly writeAnyValue = (key?: string | undefined, value?: unknown | null | undefined): void => {
```

Comment:
the conditions should be reviewed to write null only when the value is null, and not when it's undefined

Reply:
should I write `undefined` when it's `undefined` or do nothing?

Reply:
@baywet